backend/flask_api.py

```diff
@@ -86,10 +85,11 @@ class User(db.Document, UserMixin):
 
 
 class EmailConverter(BaseConverter):
-    regex = r"([a-z0-9!#$%&'*+\/=?^_`{|}~-]\
-    +(?:\.[a-z0-9!#$%&'*+\/=?^_`""{|}~-]+)\
-    *(@|\sat\s)(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])\
-    ?(\.|""\sdot\s))+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?)"
+    regex = r"([a-z0-9!#$%&'*+\/=?^_`{|}~-]+(?:\."+\
+            "[a-z0-9!#$%&'*+\/=?^_`""{|}~-]+)"+\
+            "*(@|\sat\s)(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?"+\
```

Comment:
I think there is no reason for the backslash here, operators should come with a single whitespace